spec/controllers/admin/targets_controller_spec.rb

```diff
@@ -0,0 +1,163 @@
+      end
+    end
+
+    describe 'POST #create' do
+      context 'saves successfuly' do
+        before do
+          post :create, target: attributes_for(:target), conference_id: conference.short_title
+        end
+
+        it 'redirects to admin target show path' do
+          expect(response).to redirect_to admin_conference_targets_path(conference_id: conference.short_title)
+        end
+
+        it 'shows success message in flash notice' do
+          expect(flash[:notice]).to match('Target successfully created.')
+        end
+
+        it 'creates new target' do
+          expect(Target.count).to eq 1
+        end
+      end
+
+      context 'save fails' do
+        before do
+          allow_any_instance_of(Target).to receive(:save).and_return(false)
+          post :create, target: attributes_for(:target), conference_id: conference.short_title
+        end
+
+        it 'renders new template' do
+          expect(response).to render_template('new')
+        end
+
+        it 'shows error in flash message' do
+          expect(flash[:error]).to match("Creating target failed: #{target.errors.full_messages.join('. ')}.")
+        end
+
+        it 'does not create new target' do
+          expect(Target.count).to eq 0
+        end
+      end
+    end
+
+    describe 'GET #edit' do
+      before { get :edit, conference_id: conference.short_title, id: target.id }
+
+      it 'renders edit template' do
+        expect(response).to render_template('edit')
+      end
+    end
+
+    describe 'PATCH #update' do
+      context 'updates successfully' do
+        before do
+          patch :update, target: attributes_for(:target, target_count: 2),
+                         conference_id: conference.short_title,
+                         id: target.id
+        end
+
+        it 'redirects to admin target show path' do
+          expect(response).to redirect_to admin_conference_targets_path(conference_id: conference.short_title)
+        end
+
+        it 'shows success message in flash notice' do
+          expect(flash[:notice]).to match('Target successfully updated.')
+        end
+
+        it 'updates the target' do
+          target.reload
+          expect(target.target_count).to eq 2
+        end
+      end
+
+      context 'update fails' do
+        before do
+          allow_any_instance_of(Target).to receive(:save).and_return(false)
+          patch :update, target: attributes_for(:target, target_count: 2),
+                         conference_id: conference.short_title,
+                         id: target.id
+        end
+
+        it 'renders edit template' do
+          expect(response).to render_template('edit')
+        end
+
+        it 'shows error in flash message' do
+          expect(flash[:error]).to match("Target update failed: #{target.errors.full_messages.join('. ')}.")
+        end
+
+        it 'does not update target' do
+          expect(target.target_count).not_to eq 2
```

Comment:
Please make that explicit.
You can define the target_count in your let() definition, so that we explicitely now the original value, and then we can check if the target.target_count still holds its original value.